services/quote-aggregator/main.py

```diff
@@ -72,16 +76,29 @@ def loan_quotes(event: CloudEvent):
 #                 logging.info(f"Error={err}")
 #                 raise HTTPException(status_code=500, detail=err.details())
 
+#         while True:
+#             time.sleep(1)
+
 # def shutdown_sub_stream(): 
 #     logging.info('Closing subscription...')
 #     app.state.close_fn_handler()
 
 # def loan_quotes(event):
+#     logging.info(f"Received event from {event.source()} which was published on {event.pubsub_name()} topic {event.topic()}: {event.data()}")
+
 #     with DaprClient() as d:
 #         try:
-#             logging.info(f"Received event from {event._source} which was published on {event._pubsub_name} topic {event._topic}")
+#             # logging.info(f"Received event from {event._source} which was published on {event._pubsub_name} topic {event._topic}: {event._data}")
+
+#             # ignore malformed events
+#             if event.data() is None:
+#                 logging.info(f"Event data is None, ignoring...")
+#                 return TopicEventResponse('success')
 
-#             quote_aggregate = json.loads(event._data['quote_aggregate'])
+#             quote_aggregate = event.data()
```

Comment:
no cloud events here, we get the data exactly as was published

Reply:
this was correct